### ISSUE TITLE
Debug data migration before eviction

### ISSUE THREAD

**Issue by marcinwadon** [Constellation-Labs/constellation#715]
After ~8hours I've seen such total eviction on the cluster:
![image](https://user-images.githubusercontent.com/4959931/71213185-314f6a80-22b3-11ea-9303-786f4a6f29be.png)

Both consensus proposals and accepted transactions should never be removed by eviction.

DoD:
Debug the data migration process and figure out why/when node doesn't remove consensus proposals or accepted transactions before eviction starts

**Comment by mchrapek** [comment]
Fixed deletion `consensus proposals`. The rest of the metrics look correct.

After 24 hours of cluster operation, I noticed that only things created by genesis are removed from the cache by eviction. Because things created by genesis skip `applySnapshot` step where node clears data.

I leave it this way because it does not affect the correct work of the cluster and it's hard to determine the moment when we could explicitly erase it from memory.
